src/components/NewspapersLines.vue

```diff
@@ -87,6 +92,18 @@ export default {
     },
   },
   methods: {
+    firstIssueYear(n) {
```

Comment:
to get year from a date objet: `d.getFullYear()` would be much simpler :D
I'd rewrite this function to either return null or d.getFullYear()
```javascript
firstIssueYear(n) {
  return n.firstIssue ? n.firstIssue.date.getFullYear() : null;
}
```